.github/dependabot.yml

```diff
@@ -85,3 +85,16 @@ updates:
     directory: /components/datadog/apps/cws/images/cws-centos7
     schedule:
       interval: daily
+
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: weekly
+    groups:
+      aws-sdk-go-v2:
+        patterns:
+          - '^github\.com\/aws\/aws-sdk-go-v2'
```

Comment:
I think patterns must be globs rather than regular expressions

Reply:
Thank you, fixed now